model_zoo/models/alexnet/model_alexnet.prototext

```diff
@@ -267,15 +301,28 @@ model {
 
   layer {
     name: "prob"
+    parents: "fc8"
     data_layout: "model_parallel"
     softmax {}
   }
 
+  layer {
+    name: "cross_entropy"
+    parents: "prob labels"
+    data_layout: "model_parallel"
+    cross_entropy {}    
+  }
+  layer {
+    name: "cross_entropy_eval"
+    parents: "cross_entropy"
+    data_layout: "model_parallel"
+    evaluation {}    
+  }
   layer {
     name: "target"
-    parents: "prob data"
+    parents: "prob labels"
```

Comment:
Ah, are the identity layers added to allow the implicit split layer to be defined?

Reply:
Yes, perhaps making them split layers would be a little clearer.